src/test/components/AppLocalizationProvider.test.js

```diff
@@ -210,8 +188,8 @@ describe('AppLocalizationProvider', () => {
 
     expect(await screen.findByText(translatedText('de'))).toBeInTheDocument();
     expect(document.documentElement).toHaveAttribute('lang', 'de');
-    expect(window.fetch).toBeCalledWith('/locales/de/app.ftl');
-    expect(window.fetch).toBeCalledWith('/locales/en-US/app.ftl');
+    expect(window.fetch).toBeCalledWith('/locales/de/app.ftl', undefined);
```

Comment:
It looks like `fetch` is always called with 2 arguments now, even if we don't specify the second argument in code, because of how `mock-fetch-jest` works (it installs another proxy on top of the mock). Therefore we need to match it too in the jest matcher.